static/search_events/style.css

```diff
@@ -0,0 +1,3 @@
+.width-90 {
+    width: 90%;
+}
```

Comment:
newline

Reply:
Thanks, fixed! I always forget to add the final blank line